apps/devhub/tests/test_views.py

```diff
@@ -2432,9 +2433,33 @@ def test_unlisted_addon_pass_validation(self, mock_sign_file):
             validation='{"notices": 2, "errors": 0, "messages": [],'
                        ' "metadata": {}, "warnings": 0}')
         assert self.addon.status == amo.STATUS_LITE  # Preliminary reviewed.
+
+        # Without the flag: should still go through manual prelim review.
         self.post()
         file_ = File.objects.all().order_by("-created")[0]
-        # The status is changed to preliminary reviewed and the file is signed.
+        # The status is unchanged and the file is not signed.
+        assert self.addon.status == amo.STATUS_LITE
+        assert file_.status == amo.STATUS_UNREVIEWED
+        assert not mock_sign_file.called
+
+    @mock.patch.object(waffle, 'flag_is_active')
```

Comment:
I'd suggest to waffle the flag by actually creating in the database. As it is if you remove this flag in a few months from now it will be hard to know if this is for this flag or another flag w/o really concentrating. It's also not easily "greppable".